snap/snapenv/snapenv.go

```diff
@@ -97,7 +97,7 @@ func userEnv(info *snap.Info, home string) map[string]string {
 	result := map[string]string{
 		"SNAP_USER_COMMON": info.UserCommonDataDir(home),
 		"SNAP_USER_DATA":   info.UserDataDir(home),
-		"XDG_RUNTIME_DIR":  info.UserXdgRuntimeDir(os.Geteuid()),
+		"XDG_RUNTIME_DIR":  info.UserXdgRuntimeDirGlobal(os.Geteuid()),
```

Comment:
Sigh, I see that the desktop part is creating this directory and one high-profile application is storing volatile data there. I need to think through this more. Adding back the Blocked tag.